wiki/common-platforms/unitree-go1.md

```diff
@@ -0,0 +1,104 @@
+---
+# Jekyll 'Front Matter' goes here. Most are set by default, and should NOT be
+# overwritten except in special circumstances. 
+# You should set the date the article was last updated like this:
+date: 2023-05-03 # YYYY-MM-DD
+# This will be displayed at the bottom of the article
+# You should set the article's title:
+title: Unitree Go1 Edu 
+# The 'title' is automatically displayed at the top of the page
+# and used in other parts of the site.
+---
+
+(Note: This section contains the latest information as of May 2023)
+
+This is an article that provides an overview of the Unitree Go1 Edu robot, including its features and capabilities. Unitree Robotics is a leading Chinese manufacturer that specializes in developing, producing, and selling high-performance quadruped robots. One of the company's primary advantages is that they offer quadruped platforms at a significantly lower cost compared to competitors like Boston Dynamics. In addition, they have announced plans to release experimental humanoid platforms in the near future.
+
+There are three versions of the Unitree Go1: Air, Pro, and Edu. The Edu model is designed for educational purposes and provides developers with access to the platform. In this article, we will focus on the capabilities of the Go1 Edu, which is a popular choice for students and researchers due to its affordability and ease of use.
+
+## Form Factor
+![Form_Factor](../../assets/images/form_factor.png)
```

Comment:
Please move the assets to the parent assets directory. Then, this line should read `![Form_Factor](assets/images/form_factor.png)`